executable/pe/export/forwarded-export.yml

```
@@ -0,0 +1,13 @@
+rule:
+  meta:
+    name: forwarded export
+    namespace: executable/pe/export
+    authors:
+      - ronnie.salomonsen@mandiant.com
+    scope: file
+    att&ck:
+      - Execution::Shared Modules [T1129]
+    examples:
+      - 76FA734236DAA023444DEC26863401DC:0x18003BD32
```

```suggestion
      - 76FA734236DAA023444DEC26863401DC
```

for `scope: file` there shouldn't be an associated address for the example.